### ISSUE TITLE
Clarity on setup up API key

### ISSUE THREAD

**Issue by brycenesbitt** [burnash/gspread#787]
As of this writing, if you run gspread without enabling an API key, the error message returned in the python console directs you to a URL on google's site.
That's handy, because it documents the project ID.

But you end up at this screen:
![image](https://user-images.githubusercontent.com/1294423/84426089-81124d00-abd7-11ea-8b38-9dfd9905efd9.png)

And it's not clear where to go from there.  The right answer seems to be "Google Sheets API", but the documentation sort of mentions "JSON".

Either way it's bumpy.
The developer console has many twisty details, and it's not so clear what's what.

**Comment by lavigne958** [comment]
Hi @brycenesbitt thank you for reporting this detail

Tried it, I disabled GoogleSheet API and GoogleDrive API. Ran GSpread and as you mentioned got a link.
I used the link and it took me directly to the Google Sheet API. with the big green button `Enable`.

It is very likely that the google cloud console changed since your opened your issue :thinking: 

Anyway a quick overview of the necessary APIs:
- Google Drive API: required in order to open files using their names
- Google Sheets API: required in order to do any operations on a Google Sheet file
  - open by key
  - get values/metadata
  - set values/metadata
  - clear etc..

In any cases you must enable the Google Sheets API in order to use GSpread
In some case (if you wish to open SpreadSheets using their name and not their link/key) you must enable Google Drive API.
This is due to the way GSpread looks for a file based on its title: it must search "through all files" in the drive and open the file that matches that name.

hopes this solves your issue, anyway remember to follow the documentation to completely setup your environment in order to use GSpread [here](https://docs.gspread.org/en/latest/oauth2.html)
